sdk/managedserviceidentity/Microsoft.Azure.Management.ManagedServiceIdentity/tests/Tests/IdentityTests.cs

```diff
@@ -15,7 +16,7 @@ public class IdentityTests : IDisposable
     {
         private string ResourceGroupName = "SDKTest";
         private string ExplicitIdentityType = "Microsoft.ManagedIdentity/userAssignedIdentities";
-        private string CanaryCentralRegion = "centraluseuap";
+        private string USWestCentralRegion = "westcentralus";
```

Comment:
How about name it to `TestRegion` then don't to rename again if it's changed in the future?

Reply:
I will update this once we release this API version to public